packages/atomic/src/components/insight/atomic-insight-interface/atomic-insight-interface.tsx

```diff
@@ -112,6 +113,11 @@ export class AtomicInsightInterface
    */
   @Prop({reflect: true}) resultsPerPage = 5;
 
+  /**
+   * Doc TODO
```

Comment:
Do not merge with this unresolved.

Reply:
Suggestion:

The value to set the [`nonce`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/nonce) attribute to on inline `script` and `style` elements generated by this component.

If your application is served with a Content Security Policy (CSP) that doesn't include the `script-src: 'unsafe-inline'` or `style-src: 'unsafe-inline'` directives, you must ensure that your application server generates a new nonce on every page load and uses the generated value to set this prop and serve the corresponding CSP response headers (i.e., `script-src 'nonce-<YOUR_GENERATED_NONCE>'` and `style-src 'nonce-<YOUR_GENERATED_NONCE>'`). Otherwise you may see console errors such as `Refused to execute inline script because it violates the following Content Security Policy directive: [...]` or `Refused to apply inline style because it violates the following Content Security Policy directive: [...]`.
```